index.html

```diff
@@ -356,7 +356,7 @@ <h3 class="mb-4 text-2xl font-semibold">Company</h3>
                     <p class="font-light text-gray-500 sm:text-lg dark:text-gray-400">Relevant for multiple users, extended & premium support.</p>
                     <div class="flex items-baseline justify-center my-8">
                         <span class="mr-2 text-5xl font-extrabold">$99</span>
-                        <span class="text-gray-500 dark:text-gray-400" dark:text-gray-400>/month</span>
+                        <span class="text-gray-500 dark:text-gray-400 dark:text-gray-400">/month</span>
```

Comment:
is also duplicated "dark:text-gray-400" is 2 times in the same class list

Reply:
Fixed! 👍